### ISSUE TITLE
V13 RC3 - Starting background hosted service

### ISSUE THREAD

**Issue by Nuklon** [umbraco/Umbraco-CMS#15302]
### Which Umbraco version are you using? (Please write the *exact* version, example: 10.1.0)

13 RC3

### Bug summary

When you start RC3:
```txt
[18:54:00 INF] Starting background hosted service for RecurringBackgroundJobHostedService`1
[18:54:00 INF] Starting background hosted service for RecurringBackgroundJobHostedService`1
[18:54:00 INF] Starting background hosted service for RecurringBackgroundJobHostedService`1
[18:54:00 INF] Starting background hosted service for RecurringBackgroundJobHostedService`1
[18:54:00 INF] Starting background hosted service for RecurringBackgroundJobHostedService`1
[18:54:00 INF] Starting background hosted service for RecurringBackgroundJobHostedService`1
[18:54:00 INF] Starting background hosted service for RecurringBackgroundJobHostedService`1
[18:54:00 INF] Starting background hosted service for RecurringBackgroundJobHostedService`1
[18:54:00 INF] Starting background hosted service for RecurringBackgroundJobHostedService`1
[18:54:00 INF] Starting background hosted service for RecurringBackgroundJobHostedService`1
[18:54:00 INF] Starting background hosted service for RecurringBackgroundJobHostedService`1
```

I thought something was wrong when booting Umbraco given so many identical prints, but [looking at the code](https://github.com/umbraco/Umbraco-CMS/blob/v13/dev/src/Umbraco.Infrastructure/BackgroundJobs/RecurringBackgroundJobHostedServiceRunner.cs#L47) this is expected.
It should print the job names itself so you can distinguish between them:
![image](https://github.com/umbraco/Umbraco-CMS/assets/23340481/265bec16-3f8b-45e0-8abc-f1ad02c3d862)

### Steps to reproduce

Start RC3

**Comment by kjac** [comment]
Hi @Nuklon,

Thanks for reporting this ❤️ you're absolutely right, it looks like a bug when in fact things are running as they should.

I'll have a look at it right away 👍

**Comment by kjac** [comment]
PR in https://github.com/umbraco/Umbraco-CMS/pull/15307

**Comment by kjac** [comment]
Fixed in #15307 😄
